tensorflow_model_analysis/eval_saved_model/load.py

```diff
@@ -134,7 +134,11 @@ def _load_and_parse_graph(self):
         else:
           raise ValueError('unrecognised suffix for metric. key was: %s' % k)
 
-      metric_ops = {}
+      metric_ops = {
+        'loss': tf.metrics.mean(
```

Comment:
How about we prefix it with the post_export_metric prefix as we do for the other post-export metrics to make it clear that TFMA is adding the metric, and also to avoid collisions, so something like:

`LOSS = _add_metric_prefix('loss')`

in `post_export_metrics/metric_keys.py`.